package.json

```diff
@@ -41,7 +41,7 @@
 	"dependencies": {
 		"cisco-vpn": "^1.0.2",
 		"conf": "^10.2.0",
-		"import-jsx": "^4.0.1",
+		"import-jsx": "^5.0.0",
```

Comment:
Is this package updated, but not used anywhere?

Also I'm not sure that this package will be used when moving to ESM as it looks like in the `create-ink-app` (https://github.com/vadimdemedes/create-ink-app) that it uses babel to transpile the code (https://github.com/vadimdemedes/create-ink-app/blob/main/templates/js/_package.json).

Reply:
I'll try to add `babel`. After that I'm not sure if there is anything more left before merging this 😊